addons/mass_mailing/static/src/js/mass_mailing_wysiwyg.js

```diff
@@ -4,12 +4,33 @@ import { loadBundle } from "@web/core/assets";
 import { Wysiwyg } from "@web_editor/js/wysiwyg/wysiwyg";
 import { closestElement } from "@web_editor/js/editor/odoo-editor/src/OdooEditor";
 import "@web_editor/js/wysiwyg/wysiwyg_iframe";
+import { useSubEnv } from "@odoo/owl";
 
 export class MassMailingWysiwyg extends Wysiwyg {
     //--------------------------------------------------------------------------
     // Public
     //--------------------------------------------------------------------------
 
+    setup() {
+        super.setup();
+        const onToggleFullscreen = this.env.onToggleFullscreen;
```

Comment:
No, I guess that's a good idea indeed, as it's already there, I was just a bit confused by the fact that it was not defined anywhere (yet).

I'm still not a huge fan of the way the override works though, but I guess it's the only / right way to do it.

I also don't really see why it's part of the env, it could be a proper hook method (a regular empty method inside "Wysiwyg"), that is extended here with a regular override (as we extend Wysiwyg).

I don't know, something feels off, but maybe it's just me :smile: